.changeset/sixty-dragons-grab.md

```diff
@@ -0,0 +1,7 @@
+---
+"wrangler": patch
+---
+
+feature: Sentry Integration
+Sentry will provide insights to errors and debugging potentially allow us 
+to catch bugs before they are even reported in GitHub issues. It is also inline with matching Wrangler 1 in functionality.
```

Comment:
There is the slight difference that Wrangler1 sends messages via a Worker proxy, yes?

Reply:
Correct, should that be mentioned here as an implementation difference?

Reply:
Perhaps not mention Wrangler 1 at all - since our approach here is not really aiming to replicate what Wrangler 1 does.

Reply:
Improved Changeset information